components/ui/alert/utils/helpers.ts

```diff
@@ -0,0 +1,87 @@
+import * as React from "react";
+
+import { Colors } from "../../types";
+
+import { AlertProps } from "./types";
+
+import clsx from "clsx";
+
+export function accentBorderColors(accentBorder?: boolean, color?: Colors) {
+  if (accentBorder) {
+    switch (color) {
+      case "blue": {
+        return "border-l-4 border-blue-400";
+      }
+      case "gray": {
+        return "border-l-4 border-gray-400";
+      }
+      case "green": {
+        return "border-l-4 border-green-400";
+      }
+      case "indigo": {
+        return "border-l-4 border-indigo-400";
+      }
+      case "pink": {
+        return "border-l-4 border-pink-400";
+      }
+      case "purple": {
+        return "border-l-4 border-purple-400";
+      }
+      case "red": {
+        return "border-l-4 border-red-400";
+      }
+      case "yellow": {
+        return "border-l-4 border-yellow-400";
+      }
+      default: {
+        return "border-l-4 border-yellow-400";
+      }
+    }
+  }
+}
+
+export function alertColors(color?: Colors) {
+  switch (color) {
+    case "blue": {
+      return "bg-blue-50 text-blue-700";
+    }
+    case "gray": {
+      return "bg-gray-50 text-gray-700";
+    }
+    case "green": {
+      return "bg-green-50 text-green-700";
+    }
+    case "indigo": {
+      return "bg-indigo-50 text-indigo-700";
+    }
+    case "pink": {
+      return "bg-pink-50 text-pink-700";
+    }
+    case "purple": {
+      return "bg-purple-50 text-purple-700";
+    }
+    case "red": {
+      return "bg-red-50 text-red-700";
+    }
+    case "yellow": {
+      return "bg-yellow-50 text-yellow-700";
+    }
+    default: {
+      return "bg-gray-50 text-gray-700";
+    }
+  }
+}
+
+export function renderAlertIcon(
+  icon?: AlertProps["icon"],
+  additionalClasses?: string,
+) {
+  if (icon) {
+    return React.createElement(icon, {
+      className: clsx("h-5 w-5", additionalClasses),
+      "aria-hidden": true,
+    });
+  }
+
+  return null;
+}
```

Comment:
As you can see in the warning messages from Codecov above, you need to test this file either directly (by writing a unit test for this file) or indirectly (by writing an integration test for the home page where this file is eventually used). Please write the tests for your new components. You can look at some examples in the existing `*.test.ts` or `*.test.tsx` files. 😁 
Thanks!

Reply:
I have added a unit test for the `helpers.ts` file.
Please update the tests accordingly when you're updating your implementations based on @resir014's comments.
Thanks!